packages/elements/examples/nextjs/app/sign-in/[[...sign-in]]/page.tsx

```diff
@@ -126,12 +126,18 @@ export default function SignInPage() {
                   className='flex flex-col gap-4 w-full'
                   name='identifier'
                 >
-                  <Label className='sr-only'>Email</Label>
-                  <Input
-                    className='bg-[rgb(12,12,12)] border-[rgb(37,37,37)] border rounded data-[invalid=true]:border-red-400 w-full placeholder-[rgb(100,100,100)] px-4 py-2'
-                    placeholder='Enter your email address'
-                  />
-                  <FieldError className='block text-red-400 font-mono w-full' />
+                  {fieldState => (
+                    <>
+                      <Label className='sr-only'>Email</Label>
+                      <Input
+                        className={`bg-[rgb(12,12,12)] border-[rgb(37,37,37)] border rounded w-full placeholder-[rgb(100,100,100)] px-4 py-2 ${
+                          fieldState === 'invalid' && 'border-red-500'
```

Comment:
[nit] Just an observation, I would actually consider this _not_ the proper usage of accessing field state. Given that we have the data attribute for styling purposes, that's probably how we would want to educate folks for basic styling.

For more advanced use cases, like animation, then we have the render prop for programmatic access.

Reply:
Yeah, agree. One should use the data attribute - but since this is our Playground and I needed a way to test it + have it used somewhere to see possible future regressions I opted to do that 👍